src/data-connect/data-connect.ts

```diff
@@ -165,4 +167,76 @@ export class DataConnect {
   ): Promise<ExecuteGraphqlResponse<GraphQlResponse>> {
     return this.client.upsertMany(tableName, variables);
   }
+
+  /**
+   * Executes a GraphQL query. The query must be defined in your Data Connect GraphQL files.
+   * Optionally, you may provide auth impersonation.
```

Comment:
```suggestion
   * Optionally, you can provide auth impersonation details. If you don't
   * specify a value for this option, the query will run with admin privileges
   * and will ignore all auth directives.
```

Here and throughout, rephrasing as needed